Austin_Marga_HW8_1.ipynb

```diff
@@ -0,0 +1,1230 @@
+{
```

Comment:
<p>This is a good point here. "experimental" and "theoretical" here actually compares two numerical methods numpy quad (actually an old FORTRAN library: https://en.wikipedia.org/wiki/QUADPACK) and the one your testing against. It's all fine just wanted to give a comment on working here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ubsuny/CP1_Calculus/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>